OracleGoldenGate/21c/README.md

```diff
@@ -0,0 +1,151 @@
+# Oracle GoldenGate 21.3 Microservices Edition on Docker
+
+Sample Docker build files to provide an installation of Oracle GoldenGate for DevOps users.
+These instructions apply to building Docker images for Oracle GoldenGate version 21c.
+
+## Contents
+
+* [Before You Start](#before-you-start)
+* [Build an Oracle GoldenGate Docker Image](#build-an-oracle-goldengate-docker-image)
+* [Running Oracle GoldenGate in a Docker container](#running-oracle-goldengate-in-a-docker-container)
+  * [Administrative Account Password](#administrative-account-password)
+  * [SSL Certificate](#ssl-certificate)
+  * [Running the Administration Client](#running-the-administration-client)
+* [Known Issues](#known-issues)
+* [License](#license)
+* [Copyright](#copyright)
+
+## Before You Start
+
+This project was tested with:
+
+* Oracle GoldenGate 21.3.0.0.0 Microservices for Oracle
+
+Support for Oracle GoldenGate Classic Architecture is not provided.
+
+**IMPORTANT:** You must download the installation binaries of Oracle GoldenGate. You only need to provide the binaries for the version you plan to install. The binaries can be downloaded from the [Oracle Technology Network](http://www.oracle.com/technetwork/middleware/goldengate/downloads/index.html). Do not decompress the Oracle GoldenGate ZIP file. The `dockerBuild.sh` script will handle that for you. You also must have Internet connectivity when building the Docker image for the package manager to perform additional software installations.
+
+All shell commands in this document assume the usage of Bash shell.
+
+For more information about Oracle GoldenGate please see the [Oracle GoldenGate 21.3 On-line Documentation](https://docs.oracle.com/en/middleware/goldengate/core/21.3/index.html).
+
+## Build an Oracle GoldenGate Docker Image
+
+Once you have downloaded the Oracle GoldenGate software, a Docker image is created using the standard Docker command line.
+A single `--build-arg` is needed to indicate the GoldenGate installer which was downloaded.
+
+```sh
+$ docker build --tag=oracle/goldengate:21.3.0.0.0 \
+                --build-arg INSTALLER=fbo_ggs_Linux_x64_Oracle_services_shiphome.zip .
+Sending build context to Docker daemon
+...
+Successfully tagged oracle/goldengate:21.3.0.0.0
+```
+
+## Running Oracle GoldenGate in a Docker container
+
+To run your Oracle GoldenGate Docker image use the `docker run` command as follows:
```

Comment:
Rephrase this as "Use the `docker run` command to create and start a container from the Oracle GoldenGate container image."